index.js

```diff
@@ -1,15 +1,32 @@
-var FluxConstant = function FluxConstant(name) {
+var FluxConstant = function FluxConstant(name, id) {
 
-    this.name = name;
+    this.id = String(id || (FluxConstant.lastId += 1));
+    this.name = String(name);
 };
 
 
+FluxConstant.lastId = 0;
+
+
 FluxConstant.prototype.toString = function () {
 
     return this.name;
 };
 
 
+FluxConstant.prototype.stringify = function () {
+
+    return '{"id":"' + this.id + '","name":"' + this.name + '"}';
+};
+
+
+FluxConstant.parse = function (payload) {
+
+    var data = JSON.parse(payload);
```

Comment:
I don't know if it's worth handling, but should be important to point out that this can `throw`